services/operator-value-parser.js

```diff
@@ -134,7 +134,7 @@ function OperatorValueParser(opts, timezone) {
         return {
           $gte: moment().subtract(match[1] - 1, 'days')
                   .startOf('day').toDate(),
-          $lte: moment().add(offsetHours, 'h').toDate()
+          $lte: moment().toDate()
```

Comment:
Can you explain why the end of the period does not take the timezone into account?